src/main/java/servlets/Calculator.java

```diff
@@ -0,0 +1,138 @@
+package servlets;
+
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.*;
+
+
+@WebServlet(
+        name = "Calculator",
+        urlPatterns = {"/calc"}
+)
+public class Calculator extends HttpServlet {
+    Stack<String>  stack = new Stack<>();
+    Stack<String>  nstack = new Stack<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4500113)